laravel.md

```diff
@@ -1,50 +0,0 @@
-# Laravel
```

Comment:
Why are you removing this file?

Reply:
I removed it to leave only mysql.md in that pull request.

Reply:
This way, after we merge this branch the laravel doc would get removed from the repository. That's not the right path.

In this PR your task is to add docs about mysql. So there is no need to modify/delete laravel. When it's not changed you won't see it on the Pull Request diff.
Now you have to bring back `laravel.md` to the previous state before we merge this branch to master